File: Python developer/step05/5.03/5.3.03.py
'''

Вам дано целое число n. Нужно определить, что число является палиндромом (слева направо и справа налево читается одинаково). 
Примеры палиндромов: 121, 1331, 222, 2, 11, 56165.

В случае, если число является палиндромом, выведите True, в ином случае - False.

Для тех, кто уже прошел уроки по строкам: сможете решить эту задачу в целых числах без приведения к строке?

Sample Input 1:

1331

Sample Output 1:

True

Sample Input 2:

-1331

Sample Output 2:

False

Sample Input 3:

30

Sample Output 3:

False

Sample Input 4:

11

Sample Output 4:

True

Sample Input 5:

123123

Sample Output 5:

False

Sample Input 6:

33

Sample Output 6:

True

Sample Input 7:

0

Sample Output 7:

True

Sample Input 8:

5445

Sample Output 8:

True

Sample Input 9:

-5445

Sample Output 9:

False

'''

n = input()
print(n == n[::-1])

File: Diving into JavaScript: For Beginners/step05/5.03/5.3.03.js
/*

Напишите программу, которая запрашивает у пользователя два набора из двух чисел и объявляет две функции: calculateAverage и compareAverages. 
Функция calculateAverage принимает два числа как параметры и возвращает их среднее арифметическое. 
Функция compareAverages принимает результаты средних значений из двух наборов чисел как параметры и выводит сообщение о том, 
какое среднее значение больше.

Sample Input 1:

7
5
9
3

Sample Output 1:

Средние значения наборов равны

Sample Input 2:

8
6
9
6

Sample Output 2:

Среднее значение второго набора больше

Sample Input 3:

13
46
2
5

Sample Output 3:

Среднее значение первого набора больше

Sample Input 4:

15
65
2
1

Sample Output 4:

Среднее значение первого набора больше

*/

let a1 = Number(prompt());
let a2 = Number(prompt());
let b1 = Number(prompt());
let b2 = Number(prompt());

function calculateAverage(a,b) {
    return (a + b) / 2;
}

function compareAverages(a,b) {
    if (a > b) {
        console.log('Среднее значение первого набора больше');
    } else if (b > a) {
        console.log('Среднее значение второго набора больше');
    } else {
        console.log('Средние значения наборов равны');
    }
}

let a_set = calculateAverage(a1, a2);
let b_set = calculateAverage(b1, b2);

compareAverages(a_set, b_set);
